demos/python_demos/whiteboard_inpainting_demo/utils/network_wrappers.py

```diff
@@ -0,0 +1,123 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import json
+import logging as log
+
+import cv2
+import numpy as np
+
+from utils.ie_tools import load_ie_model
+from .segm_postprocess import postprocess
+
+
+class MaskRCNN:
+    """Wrapper class for a network returning masks of objects"""
+
+    def __init__(self, ie, model_path, conf=.6, device='CPU', ext_path='',
+                 max_reqs=100):
+        self.max_reqs = max_reqs
+        self.confidence = conf
+        self.net = load_ie_model(ie, model_path, device, None, ext_path, num_reqs=self.max_reqs)
+
+        required_input_keys = [{'im_info', 'im_data'}, {'im_data', 'im_info'}]
```

Comment:
Sets are unordered.

Reply:
So, there are two options with both sequences.

Reply:
"Unordered" means that order does not matter. Following this definition, these two options are the same.

This being said, the following code should do the job:
```python
required_input_keys = {'im_info', 'im_data'}
current_input_keys = set(self.net.inputs_info.keys())
assert current_input_keys == required_input_keys
```

Reply:
Fixed.